docs/csharp/codedoc.md

```diff
@@ -12,16 +12,464 @@ ms.devlang: dotnet
 ms.assetid: 8e75e317-4a55-45f2-a866-e76124171838
 ---
 
-# 🔧 Documenting your code
```

Comment:
You should also update metadata at the start of the file: `author` should be your GitHub username, `ms.date` should be the date when the article was last updated.